sdk/storage/azure_storage_blob/tests/blob_service_client.rs

```diff
@@ -122,3 +125,46 @@ async fn test_list_containers_with_continuation(ctx: TestContext) -> Result<(),
 
     Ok(())
 }
+
+#[recorded::test]
+async fn test_get_service_stats(ctx: TestContext) -> Result<(), Box<dyn Error>> {
+    //TODO: Need playback only, need secondary-enabled GRS account to record against
+    // Recording Setup
+    let recording = ctx.recording();
+    let service_client = get_blob_service_client(recording)?;
+
+    let response = service_client.get_service_stats(None).await?;
+
+    // Assert
+    let storage_service_stats = response.into_body().await?;
+    assert!(storage_service_stats.geo_replication.is_some());
+
+    Ok(())
+}
+
+#[recorded::test]
+async fn test_get_user_delegation_key(ctx: TestContext) -> Result<(), Box<dyn Error>> {
+    let recording = ctx.recording();
+    let service_client = get_blob_service_client(recording)?;
+
+    let start = Some(format_time_string(&OffsetDateTime::now_utc()));
+    let expiry = Some(format_time_string(
+        &(OffsetDateTime::now_utc() + Duration::minutes(5)),
+    ));
+    // Act
+    let key_info = KeyInfo {
+        start: start.clone(),
+        expiry: expiry.clone(),
+    };
+
+    let response = service_client
+        .get_user_delegation_key(RequestContent::try_from(key_info)?, None)
+        .await?;
```

Comment:
Usage of this currently kind of sucks (you have to hand author the model and ensure you are using the `format_time_string` helper to get the right level of truncation on your `OffsetDateTime` resulting string).